staging/kube-oidc-proxy/templates/certificate.yaml

```diff
@@ -1,11 +1,10 @@
-{{- if .Values.tls.enabled }}
 apiVersion: certmanager.k8s.io/v1alpha1
 kind: Certificate
 metadata:
   name: kube-oidc-proxy
   namespace: {{ .Release.Namespace }}
 spec:
-  secretName: kube-oidc-proxy-kubeaddons-server-tls
+  secretName: {{ include "kube-oidc-proxy.fullname" . }}-server-tls
```

Comment:
is this backward compatible for existing installations?

Reply:
@mhrabovcin yes, i tested with an old verison, and perform an `helm upgrade kube-oidc-proxy .` and observed no change.

Reply:
We need tests to automate backwards compatibility. *sigh*